apps/src/code-studio/initApp/loadApp.js

```diff
@@ -331,11 +316,19 @@ function loadAppAsync(appOptions) {
           progress.refreshStageProgress();
         }
 
-        const signedOutUser = Object.keys(data).length === 0;
-        if (!signedOutUser && (data.disablePostMilestone ||
-                               experiments.isEnabled('postMilestoneDisabledUI'))) {
-          getStore().dispatch(disableBubbleColors());
-          showDisabledButtonsAlert(!!data.isHoc);
+        const store = getStore();
+        const signInState = store.getState().progress.signInState;
+        if (signInState === SignInState.Unknown) {
```

Comment:
Should this be checking whether `signInState` has changed, not whether it's unknown?  If we're caching this locally, what happens to a user that signs in, plays a few levels, then signs out again - does the signed-in state stay cached?